src/content/release-notes.mdx

```diff
@@ -5,6 +5,43 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.19.0
+
+16 April 2024
+
+This versions is compatible with Kubernetes versions 1.25 to 1.29
+
+### New Features
+
+- Okteto Insights: Pod v2, User, Namespace, and Preview metrics are now available <!-- 7897, 7906, 7898, 7900 -->
+
+### Breaking Changes
+
+- Okteto CLI: `okteto deploy` will now ignore host volumes. To manage the mounting of volumes, use a Dockerfile with the `COPY` instruction instead of references in your Docker compose.
+
+### Improvements
+
+- Okteto CLI: [Smart Builds implementation](core/build-service.mdx) now uses `build: context` instead of `project: commit`
```

Comment:
@codyjlandstrom link is ready:
https://www.okteto.com/docs/1.19/core/build-service/#smart-builds